lib/Dimension.js

```diff
@@ -45,11 +47,21 @@ class Dimension extends Node {
   }
 
   get cubeDimensions () {
-    return this.ptr.out(ns.view.from).out(ns.view.path).in(ns.sh.path).map(ptr => new CubeDimension({
```

Comment:
Sometimes, several shapes can be in memory, which in certain conditions, results in zero cubeDimensions.